core-bundle/src/DataContainer/PaletteManipulator.php

```diff
@@ -146,9 +146,6 @@ public function applyToString(string $palette, bool $skipLegends = false): strin
         return $this->implode($config);
     }
 
-    /**
-     * @throws PalettePositionException
-     */
```

Comment:
IMO this should not be removed (as I would not exempt private methods from the rule for `@throws`).

And `addLegend()` needs to get an `@throws PalettePositionException` too.

Reply:
Added in 4b170582f71bd8906902e2b5fd5ef2205966ae50.